controller/controller_server/csi_controller_server.py

```diff
@@ -454,15 +476,16 @@ def DeleteSnapshot(self, request, context):
         secrets = request.secrets
         try:
             utils.validate_delete_snapshot_request(request)
-            user, password, array_addresses = utils.get_array_connection_info_from_secret(secrets)
             try:
-                array_type, snapshot_id = utils.get_snapshot_id_info(request.snapshot_id)
+                volume_id_info = utils.get_snapshot_id_info(request.snapshot_id)
```

Comment:
perhaps this class name and instances should instead be named "*object* id info", sorry for the double work.
it can be in a separate ticket, if it helps

Reply:
done